collect_app/src/main/java/org/odk/collect/android/instancemanagement/InstancesDataService.kt

```diff
@@ -241,6 +241,16 @@ class InstancesDataService(
             instanceSubmitScheduler.scheduleAutoSend(projectId)
         }
     }
+
+    fun clone(instanceFile: File?, projectId: String): Long? {
```

Comment:
I'm guessing there isn't a good way to access the instance DB id in the hierarchy so we can use that instead of the path here?

Reply:
It would be more complex. Why would that be better (if it wasn't complex)?